syntax = "proto2";

package yolov3;

message UpSample {
    // layer args
    required int32 factor = 1;

    // layer attributes
    required int32 id = 2;
    optional int32 input_id = 3 [default = -1];
};

message UpSampleLayer {
    required UpSample upsample_layer = 1;
};
